packages/console/src/history.ts

```diff
@@ -18,7 +18,7 @@ export interface IConsoleHistory extends IDisposable {
   /**
    * The session context used by the foreign handler.
    */
-  readonly sessionContext: ISessionContext;
+  readonly sessionContext: ISessionContext | null;
```

Comment:
This may require people using this interface to start putting in null checks, so might break people's existing code. I think the risk is slight, but should be flagged for people?

Reply:
Yes, for now it's mentioned in the "breaking changes" section of the top PR comment. But would eventually go to the changelog too. Or did you mean mentioning the `null` change somewhere else like in the docstring of the attribute?

We can also add this PR to the 4.0 milestone just to be sure.